src/utils/token.ts

```diff
@@ -0,0 +1,3 @@
+import dayjs from 'dayjs';
+
+export const getTokenExpiresTime = () => dayjs().add(48, 'h').unix();
```

Comment:
Ten czas powinien być dłuższy niż czas ważności tokenu, bo refresh nie będzie ważnym tokenem bo wygaśnie w tym samym czasie.

Reply:
hmm to jaki powinien być ten czas?

Reply:
Myślę, że 2 razy dłuższy niż czas tokenu logowania. Żeby po tym jak user wróci do apki móc odświeżyć token logowania bez ponownego logowania